docs/README.md

```diff
@@ -5,33 +5,39 @@
 </p>
 
 # Monster Hunter Now Events
+## Table Of Contents
+* Calendar Subscription (Purpose)
+  - Steps to Subscribe Event Calendar <br>
+* Local Implementation of Calendar Subscription <br>
+* Tools <br>
+* References 
 
-This is a calendar subscription file for events in [Monster Hunter Now](https://monsterhunternow.com/).
+## Calendar Subscription
+This is a calendar subscription page for events in [Monster Hunter Now](https://monsterhunternow.com/).
 
-If you aren't playing yet, [feel free to join](https://apps.apple.com/us/app/monster-hunter-now/id6445906110) and use my referral code for free items and zenny, `89K5MKCF`.
+In case you are new to Events, [Feel free to explore and join the enriching experience.](https://apps.apple.com/us/app/monster-hunter-now/id6445906110). Use the below referral code for free items and zenny, *`89K5MKCF`*.
```

Comment:
This also is the wrong way to state what I'm going for. I'm asking folks to join and play the game with me and use my referral code. This sentence feels like it doesn't quite get across the right message.